sentry-rails/lib/sentry/rails/capture_exceptions.rb

```diff
@@ -17,7 +17,7 @@ def collect_exception(env)
       end
 
       def transaction_op
-        "rails.request".freeze
+        "http.server".freeze
```

Comment:
Breaking change? Made us reach the transaction limit 💸

Reply:
Same here...